Core/include/Acts/Propagator/detail/ParameterTraits.hpp

```diff
@@ -0,0 +1,42 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/EventData/detail/CorrectedTransformationFreeToBound.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+
+#include <tuple>
+#include <type_traits>
+
+namespace Acts::detail {
+
+template <typename stepper_t>
+struct stepper_bound_parameters {
```

Comment:
alternatively we could force the stepper to provide a `BoundType` or something like this, no? is there a benefit of having a trait?

Reply:
So the trait does not change the requirements on the steppers, and thus wouldn't break custom steppers...